master/reference/calicoctl/resources/bgpconfig.md

```diff
@@ -0,0 +1,47 @@
+---
+title: BGP Configuration Resource (BGPConfiguration)
+---
+
+A BGP configuration resource (BGPConfiguration) represents BGP specific configuration options for the cluster.
+
+For `calicoctl` commands that specify a resource type on the CLI, the following
+aliases are supported (all case insensitive): `bgpconfiguration`, `bgpconfig`, `bgpconfigurations`, `bgpconfigs`.
+
+### Sample YAML
+
+```yaml
+apiVersion: projectcalico.org/v2
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  logSeverityScreen: info
+  nodeToNodeMeshEnabled: true
+  asNumber: 63400
+```
+
+### BGP Peer Definition
+
+#### Metadata
+
+| Field       | Description                 | Accepted Values   | Schema |
+|-------------|-----------------------------|-------------------|--------|
+| name     | Unique name to describe this resource instance. Must be specified. | Alphanumeric string with optional ".", "_", "-", or "/" | string |
+
+- Resource with the name "default" has a specific meaning - this contains the BGP global default configuration overrides.
```

Comment:
This sentence is a little vague, has a few minor grammatical issues, and seems to contain some unnecessary words. I am not exactly sure what you are trying to say, but I think it could be the following: "Resources named `default` override BGP global default configuration settings."

Of course, it does make me wonder what happens if two resources are named `default` and override the same global default configuration setting with different values: which takes precedence? But maybe I am going too much into the twigs. Or maybe only one resource can be named `default`. If it is the latter, then I would suggest adding this information.

Reply:
that sentence had an extra "overrides" at the end, but this is just trying to say that a BGPConfig resource with the name of `default` is considered the global BGP config, there can only be one resource with that name. Let me know if you have a different/more readable sentence